content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -383,6 +383,14 @@ This will remove the `node-role.kubernetes.io/control-plane:NoSchedule` taint
 from any nodes that have it, including the control plane nodes, meaning that the
 scheduler will then be able to schedule Pods everywhere.
 
+Additionally, you can execute the following command to remove the
+"node.kubernetes.io/exclude-from-external-load-balancers" label
```

Comment:
Consider using Markdown code highlighting syntax instead of quotes for an improved presentation.

```suggestion
`node.kubernetes.io/exclude-from-external-load-balancers` label
```

Reply:
also, you can make this label link to its info at the "known labels" page. my example on the issue did that.

Reply:
I have now made the suggested changes. Please review again. THanks!